lib/util/string.ts

```diff
@@ -55,3 +55,7 @@ export function looseEquals(
   }
   return a.localeCompare(b, undefined, { sensitivity: 'base' }) === 0;
 }
+
+export function isDockerDigest(input: string): boolean {
+  return /^sha256:[a-f0-9]{64}$/i.test(input);
```

Comment:
why not using re2 ?

Reply:
Seems like it would be too soon:

```
$ node -r ts-node/register/transpile-only -- tools/generate-schema.ts && prettier --write --cache "renovate-schema.json"
/home/runner/work/renovate/renovate/lib/util/regex.ts:18
  logger.warn({ err }, 'RE2 not usable, falling back to RegExp');
         ^
TypeError: Cannot read property 'warn' of undefined
```

Reply:
That sounds strange

Reply:
I think string util is used before logger was initialized

Reply:
Maybe put it back in the same file then?

Reply:
it's probably also ok to use plain regex here, as we don't have any risky regex features here.